### ISSUE TITLE
Padded Feet doesn't work with fur

### ISSUE THREAD

**Issue by KittyTac** [CleverRaven/Cataclysm-DDA#58298]
### Describe the bug

Padded feet doesn't give a 10% movement boost if barefoot but with fur. This is because integrated fur armor counts for its purposes.

### Steps to reproduce

1. Give your character fur.
2. Give your character tough feet, walk a tile and note down the move cost.
3. Give your character padded feet and see that the move cost didn't change.

### Expected behavior

For the movement cost to get reduced by 10%. The padded feet mutation should not be blocked by integrated armor.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.18363.1256 (1909)
- Game Version: befb7ab [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Extra Mutated Scenarios [extra_mut_scens],
    No Rail Stations [No_Rail_Stations]
]

### Additional context

N/A

**Comment by Amoebka** [comment]
Duplicate of 56919 for anyone interested.

**Comment by anothersimulacrum** [comment]
Closing as duplicate of https://github.com/CleverRaven/Cataclysm-DDA/issues/58298
